libs/libc/pthread/pthread_atfork.c

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * libs/libc/pthread/pthread_atfork.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <pthread.h>
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+int pthread_atfork(CODE void (*prepare)(void),
+                   CODE void (*parent)(void),
+                   CODE void (*child)(void))
+{
+  /* fork isn't supported yet, so the dummy implementation is enough. */
+
+  return 0;
```

Comment:
```suggestion
  return ENOSYS;
```
Wouldn't it be better to implement it as a stub returning `ENOSYS` to indicate it is not implemented?

Reply:
@gustavonihei all callbacks should never get called on NuttX because fork isn't implemented. So, it's the right implementation even without any real action.